### PR TITLE
Feature: adds /digests handling

### PR DESCRIPTION
**Pull request by lsh-0**
ticket: https://github.com/elifesciences/issues/issues/4290

there are no published digests yet and these digests don't appear to relate to the digests of yore, with those either living on Medium or incorporated into the article.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ boto3==1.5.22
 coverage==4.5
 cryptography==2.1.4
 decorator==4.2.1
-Django==1.11.10
+Django==1.11.15
```
Comment (lsh-0) [inline requirements.txt]: these changes to requirements.txt are redundant, I have another PR open that is bumping them right now

File: schema/metrics/history.json [hunk 2]
```diff
@@ -139,5 +139,16 @@
...
             }
         ]
+    },
+
+    "digests": {
+        "frames": [
+            {
+                "id": 1,
+                "ends": null,
+                "starts": "2017-09-01",
```
Comment (lsh-0) [inline schema/metrics/history.json]: this is the 'digests started appearing on the webiste' date and not a published date.

Comment (giorgiosironi) [reply schema/metrics/history.json]: Correct me if I'm wrong but the metrics of:

- a digest published on 2017-09-01 and viewed 0 times on 2017-09-01, 42 times on 2017-09-02
- a digest published on 2017-09-02 and viewed 42 times on 2017-09-02

are equivalent, right?

Comment (giorgiosironi) [reply schema/metrics/history.json]: (It's a mostly unrelated question)

**Comment by giorgiosironi** [comment]
(needs a conflict resolution)

File: schema/metrics/history.json (hunk 2 above)
Comment (lsh-0) [reply schema/metrics/history.json]: ... they are equivalent in terms of total page views, as the dates are inclusive. Did I understand you correctly?

this file is used to describe 'epochs' of url patterns. 2017-09-01 is arbitrary, it's simply a nice round value that happened before digests were/will be introduced. I've gone to some lengths over time to try and capture all the different permutations we've seen. Article metrics are still their own little world but I'd like to migrate them into this file as well

**Pull request merged**